autometa/common/external/samtools.py

```diff
@@ -74,9 +74,8 @@ def sort(sam, bam, cpus=mp.cpu_count()):
                     cmd, stdout=stdout, stderr=stderr, shell=True, check=True
                 )
             except subprocess.CalledProcessError as err:
-                err.cmd = cmd
-                logger.error(f"SamtoolsSortError: {cmd}")
-                raise SamtoolsSortError(err)
+                logger.error(f"Error while running samtools \n cmd: {cmd}")
+                raise ExternalToolError(cmd, err)
```

Comment:
I think it may be more appropriate to log this exception when handling it.

```suggestion
                raise ExternalToolError(cmd, err)
```